### PR TITLE
improve pub score

### PR DESCRIPTION
**Pull request by felangel**
Addressed the following:
screen shot 2018-11-25 at 12 29 16 am

### CODE REVIEW

File: example/example.dart [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
```
Comment (grouma) [inline example/example.dart]: [nit] 2018

File: lib/src/copy/web_socket_impl.dart [hunk 2]
```diff
@@ -23,13 +23,8 @@ import 'io_sink.dart';
 import 'web_socket.dart';
 
 const String webSocketGUID = "258EAFA5-E914-47DA-95CA-C5AB0DC85B11";
-const String _webSocketGUID = "258EAFA5-E914-47DA-95CA-C5AB0DC85B11";
```
Comment (grouma) [inline lib/src/copy/web_socket_impl.dart]: Odd these weren't cleaned up / caught sooner. @kevmoo do we not have the proper lints turned on for this package?

File: example/example.dart (hunk 1 above)
Comment (felangel) [reply example/example.dart]: sorry I copied the file header from CONTRIBUTING. I can update both now.

File: lib/src/copy/web_socket_impl.dart (hunk 2 above)
Comment (felangel) [reply lib/src/copy/web_socket_impl.dart]: FYI ```dartanalyzer --fatal-infos --fatal-warnings .``` caught these

Comment (grouma) [reply lib/src/copy/web_socket_impl.dart]: Ah yes. That's our usual standard. Thanks for fixing it.

**Pull request merged**